lowell-dashboard/app/config.py

```diff
@@ -3,13 +3,15 @@
 Ex: database configurations, keys, etc.
 '''
 
+import secret
```

Comment:
i think it has to be from app import secret since it's in the app folder.

Reply:
In the same folder as config folder